src/device/wg_controller.rs

```diff
@@ -247,6 +257,34 @@ async fn consolidate_wg_peers<
 
             (_, _) => {}
         }
+        telio_log_debug!(
+            "peer {:?} proxying: {:?}, state: {:?}, lh: {:?}",
+            actual_peer.public_key,
+            is_actual_peer_proxying,
+            actual_peer.state(),
+            actual_peer.time_since_last_handshake
+        );
+        if is_actual_peer_proxying && actual_peer.state() == Connected {
+            is_any_peer_eligible_for_upgrade = true;
+        }
+    }
+
+    let ep_control = |ep: Arc<dyn EndpointProvider>| async move {
+        if is_any_peer_eligible_for_upgrade {
+            telio_log_debug!("Unpausing {} provider", ep.name());
```

Comment:
Could you make these logs `Info` level?

Reply:
it is spamming every 5 seconds, not sure if this is needed.